README.md

```
@@ -28,21 +38,59 @@ the expected `PCR0`. A client that skips verification gets no guarantee.
 
 ## Architecture
 
+There are **two inbound paths and they do not have the same trust properties.**
+
+### A. Public path — `https://enclave.ppq.ai` (web app + npm package)
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)